CHANGELOG.md

```diff
@@ -6,6 +6,8 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 ### Added
+- `BayesianGaussianMixture` model
+- `serialize_tuple` method in `general_data_structure_transporter.py`
```

Comment:
Isn't it better like this?:
```
`serialize_tuple` method to `GeneralDataStructureTransporter` class
```

Reply:
I convert it to `serialize_tuple` method in `GeneralDataStructureTransporter` to make it compatible with similar previous ones.
@sadrasabouri